nebula_ros/src/continental/continental_ars548_ros_wrapper.cpp

```diff
@@ -0,0 +1,352 @@
+    config.configuration_vehicle_length =
+      static_cast<float>(this->get_parameter("configuration_vehicle_length").as_double());
+  }
+  {
+    rcl_interfaces::msg::ParameterDescriptor descriptor = param_read_write();
+    descriptor.type = rcl_interfaces::msg::ParameterType::PARAMETER_DOUBLE;
+    descriptor.read_only = true;
+    descriptor.dynamic_typing = false;
+    descriptor.additional_constraints = "";
+    this->declare_parameter<double>("configuration_vehicle_width", descriptor);
+    config.configuration_vehicle_width =
+      static_cast<float>(this->get_parameter("configuration_vehicle_width").as_double());
+  }
+  {
+    rcl_interfaces::msg::ParameterDescriptor descriptor = param_read_write();
+    descriptor.type = rcl_interfaces::msg::ParameterType::PARAMETER_DOUBLE;
+    descriptor.read_only = true;
+    descriptor.dynamic_typing = false;
+    descriptor.additional_constraints = "";
+    this->declare_parameter<double>("configuration_vehicle_height", descriptor);
+    config.configuration_vehicle_height =
+      static_cast<float>(this->get_parameter("configuration_vehicle_height").as_double());
+  }
+  {
+    rcl_interfaces::msg::ParameterDescriptor descriptor = param_read_write();
+    descriptor.type = rcl_interfaces::msg::ParameterType::PARAMETER_DOUBLE;
+    descriptor.read_only = true;
+    descriptor.dynamic_typing = false;
+    descriptor.additional_constraints = "";
+    this->declare_parameter<double>("configuration_vehicle_wheelbase", descriptor);
+    config.configuration_vehicle_wheelbase =
+      static_cast<float>(this->get_parameter("configuration_vehicle_wheelbase").as_double());
+  }
+
+  if (config.sensor_model == nebula::drivers::SensorModel::UNKNOWN) {
+    return Status::INVALID_SENSOR_MODEL;
+  }
+
+  auto new_config_ptr = std::make_shared<
+    const nebula::drivers::continental_ars548::ContinentalArs548SensorConfiguration>(config);
+  return ValidateAndSetConfig(new_config_ptr);
+}
+
+Status ContinentalArs548RosWrapper::ValidateAndSetConfig(
+  std::shared_ptr<const drivers::continental_ars548::ContinentalArs548SensorConfiguration> &
+    new_config_ptr)
+{
+  if (new_config_ptr->sensor_model == nebula::drivers::SensorModel::UNKNOWN) {
+    return Status::INVALID_SENSOR_MODEL;
+  }
+
+  if (new_config_ptr->frame_id.empty()) {
+    return Status::SENSOR_CONFIG_ERROR;
+  }
+
+  if (hw_interface_wrapper_) {
+    hw_interface_wrapper_->OnConfigChange(new_config_ptr);
+  }
+  if (decoder_wrapper_) {
+    decoder_wrapper_->OnConfigChange(new_config_ptr);
+  }
+
+  config_ptr_ = new_config_ptr;
+  return Status::OK;
+}
+
+void ContinentalArs548RosWrapper::ReceivePacketsMessageCallback(
+  std::unique_ptr<nebula_msgs::msg::NebulaPackets> packets_msg)
+{
+  if (hw_interface_wrapper_) {
+    RCLCPP_ERROR_THROTTLE(
+      get_logger(), *get_clock(), 1000,
+      "Ignoring NebulaPackets PandarScan. Launch with launch_hw:=false to enable NebulaPackets "
```

Comment:
Stray PandarScan?

Reply:
And I just commented the same to Max :sweat_smile: 
Fixed in [2a1e08b](https://github.com/tier4/nebula/pull/151/commits/2a1e08b92b4a4dfbde88094479475ce56d85d5ef)